_includes/code/howto/manage-data.import.py

```diff
@@ -49,6 +49,13 @@
             properties=data_row,
         )
 # highlight-end
+        if batch.number_errors > 0:
+            print(f"Error found during batch import")
+
+failed_objects = collection.batch.failed_objects
+if failed_objects:
+    print(f"Number of failed imports: {len(failed_objects)}")
```

Comment:
At first, I was going to propose to use `collection .batch.number_errors` instead of `len(failed_objects)`.
But I actually like it, as it feels clean.